content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -7,31 +7,36 @@ slug: dockershim-faq
 aliases: [ '/dockershim' ]
 ---
 
-**This is an update to the original [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article,
-published in late 2020.**
+**This supersedes the original
+[Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article,
+published in late 2020. The article includes updates from the v1.24
+release of Kubernetes.**
 
 This document goes over some frequently asked questions regarding the
-deprecation and removal of _dockershim_, that was
+removal of _dockershim_ from Kubernetes. That removal was originally
 [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/)
-as a part of the Kubernetes v1.20 release. For more detail
-on what that means, check out the blog post
+as a part of the Kubernetes v1.20 release. The Kubernetes
+[v1.24 release](/releases/#release-v1-24) actually removed the dockershim
```

Comment:
```suggestion
[v1.24 release](/releases/#release-v1-24) completely removed the dockershim
```
or "fully removed" or "finalized the removal of"

Reply:
There was no partial removal; I'm worried that this implies that there was.

Reply:
Yeah and this is fine as-is, thanks :)